### PR TITLE
Adapt System UI colour in light and dark themes (EXPOSUREAPP-6991)

### PR DESCRIPTION
**Pull request by mtwalli**
System UI: 
1- Status bar 
2- Navigation  bar 

## Dark
Screenshot 2021-05-05 at 11 24 19

## Light
Screenshot 2021-05-05 at 11 23 09

### CODE REVIEW

**Comment by dsarkar** [comment]
fixes https://github.com/corona-warn-app/cwa-app-android/issues/3041

File: Corona-Warn-App/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -11,6 +11,13 @@
         <item name="materialTimePickerTheme">@style/ThemeOverlay.App.TimePicker</item>
 
         <item name="buttonStyle">@style/Widget.AppCompat.Button.Borderless.Colored</item>
+
+        <item name="android:windowLightNavigationBar" tools:targetApi="o_mr1">@bool/lightSystemUI
+        </item>
```
Comment (d4rken) [inline Corona-Warn-App/src/main/res/values/styles.xml:16]: :fearful:

**Review by d4rken — APPROVED** [review]
:+1:

**Review by LukasLechnerDev — APPROVED** [review]
LGTM! Tested on Samsung S20+

**Comment by binarynoise** [comment]
Closed #3041

**Pull request merged**